### DISCUSSION TITLE
Contribution Activity Graph Color

### DISCUSSION THREAD

**Discussion by PickleNik** [community/community#42327] [Other Feature Feedback, Questions, & Ideas]

**Comment by ghostinhershell** [comment]
Hey there!

Thanks for posting in the GitHub Community, @PickleNik! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by p3rc1us** [comment]
hello, i think theres a bug with my contribution highlights. Most of it must be more vibrant green rather than uniformly less colored. I believe it is working properly several days ago but since yesterday almost all of it is uniformly less colored.
